libbpf-tools/trace_helpers.c

```diff
@@ -256,6 +256,11 @@ static bool is_vdso(const char *path)
 	return !strcmp(path, "[vdso]");
 }
 
+static bool is_uprobes(const char *path)
+{
+	return !strcmp(path, "[uprobes]");
+}
+
```

Comment:
is_file_backed also need this.

Reply:
Thanks for reviewing, is_file_backed is patched.
A simple reproducer is also added to commit message.